vitessce/entities.py

```diff
@@ -151,4 +155,100 @@ def add_molecule(self, name, coords):
     :param str name: Name for the new molecules
     :param list coords: A list of lists like [[1, 2], [3, 4], ...] or [[1, 2, 3], [3, 4, 5], ...] which denote where in xy space the spot data should be placed for the desired name.
     """
-    self.json[name] = coords
\ No newline at end of file
+    self.json[name] = coords
+
+class GenomicProfiles():
+  
+  """
+  Generic class for representing genomic profiles.
+  """
+
+  def __init__(self, f, profile_ids, assembly='hg38', starting_resolution=5000, name="Genomic Profiles"):
```

Comment:
Can we add doc strings for these methods?

Reply:
Oh yes sorry about that. I just added some